index.d.ts

```diff
@@ -39,6 +39,13 @@ declare namespace globby {
 		@default false
 		*/
 		readonly gitignore?: boolean;
+
+		/**
+		Glob pattern to look for ignore files, which are then used to ignore globbed files.
+
+		@default undefined
```

Comment:
This is not correct.

Reply:
Can you explain? If `ignoreFiles` isn't specified, we don't do anything with the value. Would you rather this say "empty string", or something else?

Reply:
Sorry, I don't remember. I should have been clearer.